### PR TITLE
Upgrade to tensorflow2

### PR DESCRIPTION
**Pull request by atorch**
* I couldn't get class weights to work, so I deleted them and replaced them with temporal sample weights
  - I should explain the idea behind masking in the readme: CDL developed contains roads, buildings, and (at 30 meter resolution) many NAIP pixels that are neither roads nor buildings, hence the idea of masking out CDL developed when computing loss.
  - Without masking, the model learns to predict developed for pixels _near_ roads.  Often, the ground truth is that these pixels are crops or pasture or forest.
* It looks like https://github.com/keras-team/keras/issues/3653 still has no elegant solution, even with tensorflow2

### CODE REVIEW

File: src/cnn.py [hunk 1]
```diff
@@ -68,26 +68,6 @@ def add_upsampling_block(input_layer, block_index, downsampling_conv2_layers):
...
-
-        mask = K.ones_like(y_true[:, :, :, 0])
-
-        for cdl_index in cdl_indices_to_mask:
-            cdl_index_indicators = K.cast(
-                K.equal(y_true[:, :, :, cdl_index], 1), "float32"
-            )
-            mask -= cdl_index_indicators
-
-        return K.categorical_crossentropy(y_true, y_pred) * mask
```
Comment (atorch) [inline src/cnn.py:86]: I got errors here when running fit_model.py on this branch

File: src/cnn.py [hunk 2]
```diff
@@ -139,21 +119,19 @@ def get_keras_model(image_shape, label_encoder):
 
     print(model.summary())
 
-    nadam = keras.optimizers.Nadam()
+    nadam = optimizers.Nadam()
 
+    # TODO Unused, reimplement this?
     cdl_indices_to_mask = label_encoder.transform(CDL_CLASSES_TO_MASK)
```
Comment (atorch) [inline src/cnn.py:139]: Explain the idea behind masking in the readme:  CDL developed contains roads, buildings, and (at 30 meter resolution) many NAIP pixels that are neither roads nor buildings, hence the idea of masking out CDL developed when computing loss.

File: src/cnn.py (hunk 1 above)
Comment (atorch) [reply src/cnn.py:86]: I replaced this with temporal sample weights (based on class):  see https://github.com/keras-team/keras/issues/3653#issuecomment-557844450
